src/db_adapters/accounts.rs

```diff
@@ -326,3 +327,44 @@ pub(crate) async fn store_accounts_from_genesis(near_config: near_indexer::NearC
         "Accounts from genesis were added/updated successful."
     );
 }
+
+pub(crate) async fn collect_lockups_for_block(
```

Comment:
```suggestion
pub(crate) async fn collect_lockups_for_block_height(
```

I love your habit to comment such cases! Amazing!